packages/babel-preset-expo/src/__tests__/use-dom-directive-plugin.test.ts

```diff
@@ -17,19 +17,27 @@ function getCaller(props: Record<string, string | boolean>): babel.TransformCall
   return props as unknown as babel.TransformCaller;
 }
 
-const DEF_OPTIONS = {
-  // Ensure this is absolute to prevent the filename from being converted to absolute and breaking CI tests.
-  filename: '/unknown',
-
-  babelrc: false,
-  presets: [[preset, { disableImportExportTransform: true }]],
-  sourceMaps: true,
-  configFile: false,
-  compact: false,
-  comments: true,
-  retainLines: false,
-  caller: getCaller({ ...ENABLED_CALLER, platform: 'ios' }),
-};
+function createBabelOptions({
+  filename,
+  presetOptions,
+}: {
+  filename?: string;
+  presetOptions?: BabelPresetExpoOptions;
+}): babel.TransformOptions {
+  return {
+    // Ensure this is absolute to prevent the filename from being converted to absolute and breaking CI tests.
+    filename: filename || '/unknown',
+
+    babelrc: false,
+    presets: [[preset, { disableImportExportTransform: true, ...presetOptions }]],
```

Comment:
allowing customized `filename` and `presetOptions` because we need filename as `*.tsx` and `disableImportExportTransform: false` to test this case